src/Microsoft.Health.Dicom.Api/Features/Filters/DicomRequestContextRouteDataPopulatingFilterAttribute.cs

```diff
@@ -39,7 +39,20 @@ public override void OnActionExecuting(ActionExecutingContext context)
         IDicomRequestContext dicomRequestContext = _dicomRequestContextAccessor.RequestContext;
         dicomRequestContext.RouteName = context.ActionDescriptor?.AttributeRouteInfo?.Name;
 
-        dicomRequestContext.Version = context.HttpContext.GetRequestedApiVersion()?.MajorVersion;
+        ApiVersion version = null;
+        try
+        {
+            version = context.HttpContext?.GetRequestedApiVersion();
```

Comment:
I also added another method that invokes this extension method internally. Can you double-check all of the places we call `GetRequestedApiVersion()`?